packages/design-system/src/Avatar/Avatar.styles.tsx

```diff
@@ -0,0 +1,75 @@
+import styled, { css } from "styled-components";
+import { AvatarClassName } from "./Avatar.constants";
+import { AvatarSize } from "./Avatar.types";
+
+const Sizes = {
+  sm: css`
+    width: 24px;
```

Comment:
is there a reason we aren't doing our usual approach of variable changing here?

Reply:
Fixed